pytorch_lightning/trainer/trainer.py

```diff
@@ -1178,10 +1178,13 @@ def _call_teardown_hook(self, model: 'pl.LightningModule') -> None:
 
         if self.datamodule is not None:
             self.datamodule.teardown(stage=fn)
+
         self.profiler.teardown(stage=fn)
         self.teardown(stage=fn)
         model.teardown(stage=fn)
 
+        self._active_loop.teardown()
```

Comment:
this `teardown` is already called in 

https://github.com/PyTorchLightning/pytorch-lightning/blob/2d682fa4618c8c70535c55c20d51e73fc367c948/pytorch_lightning/trainer/trainer.py#L961-L966

Reply:
Removed !